File: output/ChatGPT/codenet/Python/Java/s238775098.java
import java.util.Scanner;

public class s238775098 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        double res = 0;
        
        for (int i = 0; i < n; i++) {
            double x = scanner.nextDouble();
            String u = scanner.next();
            
            if (u.equals("BTC")) {
                res += x * 380000;
            } else {
                res += x;
            }
        }
        
        System.out.println(res);
    }
}

File: output/ChatGPT/codenet/Python/C++/s238775098.cpp
#include <iostream>
#include <string>

int main() {
    int n;
    std::cin >> n;
    double res = 0;
    for (int i = 0; i < n; i++) {
        std::string x, u;
        std::cin >> x >> u;
        if (u == "BTC") {
            res += std::stod(x) * 380000;
        } else {
            res += std::stoi(x);
        }
    }
    std::cout << res << std::endl;

    return 0;
}